ext/CEGUI/IrrlichtBaWRenderTarget.cpp

```diff
@@ -1,30 +1,47 @@
 #include "IrrlichtBaWRenderTarget.hpp"
+#include "IrrlichtBaWGeometryBuffer.hpp"
 
 using namespace irr;
 using namespace ext;
 using namespace cegui;
 
+const double IrrlichtBaWRenderTarget::d_yfov_tan = 0.267949192431123;
```

Comment:
a GUI projection is orthographic, not perspective

Reply:
it is copied from their opengl implementation

Reply:
well in that case its wrong XD